internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -365,26 +364,26 @@ func TestAccClusterAdvancedCluster_advancedConf(t *testing.T) {
 		processArgs            = &matlas.ProcessArgs{
 			DefaultReadConcern:               "available",
 			DefaultWriteConcern:              "1",
-			FailIndexKeyTooLong:              pointy.Bool(false),
-			JavascriptEnabled:                pointy.Bool(true),
+			FailIndexKeyTooLong:              conversion.Pointer(false),
+			JavascriptEnabled:                conversion.Pointer(true),
 			MinimumEnabledTLSProtocol:        "TLS1_1",
-			NoTableScan:                      pointy.Bool(false),
-			OplogSizeMB:                      pointy.Int64(1000),
-			SampleRefreshIntervalBIConnector: pointy.Int64(310),
-			SampleSizeBIConnector:            pointy.Int64(110),
-			TransactionLifetimeLimitSeconds:  pointy.Int64(300),
+			NoTableScan:                      conversion.Pointer(false),
+			OplogSizeMB:                      conversion.Pointer[int64](1000),
```

Comment:
does it fail if [int64] is not specified?

Reply:
yes, OplogSizeMB is int64 and the literal `1000` is int so you need to provide the cast